channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -622,6 +649,49 @@ export class Sender extends BaseTelemetryPlugin implements IChannelControlsAI {
         
                 _self._buffer.markAsSent(payload);
             }
+
+            /**
+             * Send fetch API request
+             * @param payload {string} - The data payload to be sent.
+             * @param isAsync {boolean} - not used
+             */
+             function _fetchSender(payload: string[], isAsync: boolean) {
+                const endPointUrl = _self._senderConfig.endpointUrl();
+                const batch = _self._buffer.batchPayloads(payload);
+                const plainTextBatch = new Blob([batch], { type: 'text/plain;charset=UTF-8' });
+                let requestHeaders = new Headers();
+                // append Sdk-Context request header only in case of breeze endpoint
+                if (isInternalApplicationInsightsEndpoint(endPointUrl)) {
+                    requestHeaders.append(RequestHeaders.sdkContextHeader, RequestHeaders.sdkContextHeaderAppIdRequest);
+                }
+                arrForEach(objKeys(_headers), (headerName) => {
+                    requestHeaders.append(headerName, _headers[headerName]);
+                });
+                const init = {
+                    method: "POST",
+                    headers: requestHeaders,
+                    body: plainTextBatch
+                }
+                const request = new Request(endPointUrl, init);
+
+                fetch(request).then(response => {
```

Comment:
As per stamp specific redirects, @MSNev, @xiao-lix  I am thinking if its ideal to automatically follow the redirect with the following fetch option? redirect: 'follow' .